bundles/binding/org.openhab.binding.octoller/src/main/java/org/openhab/binding/octoller/internal/octollerGenericBindingProvider.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */ 
+
+package org.openhab.binding.octoller.internal;
+
+import org.openhab.binding.octoller.octollerBindingProvider;
+import org.openhab.core.binding.BindingConfig;
+import org.openhab.core.items.Item;
+import org.openhab.core.library.items.DimmerItem;
+import org.openhab.core.library.items.SwitchItem;
+import org.openhab.model.item.binding.AbstractGenericBindingProvider;
+import org.openhab.model.item.binding.BindingConfigParseException;
+
+
+/**
+ * This class is responsible for parsing the binding configuration.
+ * For openHAB binding for octoller (www.octoller.com)
```

Comment:
remove generic comment and briefly explain what this class is about, also add some Binding configuration examples here.